package.json

```diff
@@ -77,6 +77,7 @@
     "cmdk": "^0.2.0",
     "drizzle-orm": "0.26.5",
     "drizzle-zod": "0.4.2",
+    "encoding": "^0.1.13",
```

Comment:
don't go for the latest version, it may cause many issues in terms of type etc. because these packages are still in rapid development

Reply:
okay sir